extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -398,6 +419,18 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
     const typeConfig = this.predefinedConfig_[type] || {};
 
+    // transport.iframe is only allowed to be specified in typeConfig, not
+    // the others. Allowed when running locally for testing purposes.
+    if (!getMode().localDev &&
+        ((defaultConfig.transport && defaultConfig.transport.iframe) ||
+         (inlineConfig && inlineConfig.transport &&
+          inlineConfig.transport.iframe) ||
+         (this.remoteConfig_ && this.remoteConfig_.transport &&
+          this.remoteConfig_.transport.iframe))) {
+      user().error(this.getName_(), 'transport/iframe may only be specified' +
```

Comment:
You need to actually return here or otherwise prevent this situation.  As is, I believe this would allow the code to continue to be executed.